src/index.js

```diff
@@ -30,6 +30,8 @@ type Props = {|
 |}
 
 class ActiveStorageProvider extends React.Component<Props> {
+  static defaultProps = { headers: {} }
```

Comment:
Re: the flow error that has popped up in semaphore

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

https://flow.org/en/docs/react/components/#toc-using-default-props